administration/configuring-fluent-bit/classic-mode/configuration-file.md

```diff
@@ -1,96 +1,107 @@
-| scheduler.cap   | Set a maximum retry time in second. The property is supported from v1.8.7.                                                                                                                                                                                                                              | 2000          |
-| scheduler.base  | Set a base of exponential backoff. The property is supported from v1.8.7.                                                                                                                                                                                                                               | 5             |
-| json.convert_nan_to_null | If enabled, NaN is converted to null when fluent-bit converts msgpack to json.    | false         |
-| sp.convert_from_str_to_num | If enabled, Stream processor converts from number string to number type.        | true          |
-
-The following is an example of a _SERVICE_ section:
+One of the ways to configure Fluent Bit is using a main configuration file. Fluent
+Bit allows the use one configuration file which works at a global scope and uses the
+defined [Format and Schema](format-schema.md).
+
+The main configuration file supports four sections:
+
+- Service
+- Input
+- Filter
+- Output
+
+It's also possible to split the main configuration file into multiple files using
+the Include File feature to include external files.
+
+## Service
+
+The `Service` section defines global properties of the service. The following keys
+are:
+
+| Key             | Description   | Default Value |
+| --------------- | ------------- | ------------- |
+| `flush`           | Set the flush time in `seconds.nanoseconds`. The engine loop uses a Flush timeout to define when it's required to flush the records ingested by input plugins through the defined output plugins.  | `1` |
+| `grace`           | Set the grace time in `seconds` as an integer value. The engine loop uses a Grace timeout to define wait time on exit. | `5` |
+| daemon          | Boolean. Set if Fluent Bit should run as a Daemon (background) or not. Allowed values are: `yes`, `no`, `on`, and `off`. When using a Systemd based unit as the one provided in FLuent Bit packages, don't turn on this option.  | `Off` |
+| `dns.mode`        | Set the primary transport layer protocol used by the asynchronous DNS resolver. Can be overridden on a per plugin basis. | `UDP` |
+| `log_file`        | Absolute path for an optional log file. By default all logs are redirected to the standard error interface (stderr). | _none_ |
+| `log_level`       | Set the logging verbosity level. Allowed values are: `off`, `error`, `warn`, `info`, `debug`, and `trace`. Values are cumulative. If 'debug' is set, it will include `error`, `warning`, `info`, and `debug`. Trace mode is only available if Fluent Bit was built with the _`WITH_TRACE`_ option enabled. | `info` |
```

Comment:
```suggestion
| `log_level`       | Set the logging verbosity level. Allowed values are: `off`, `error`, `warn`, `info`, `debug`, and `trace`. Values are cumulative. If `debug` is set, it will include `error`, `warning`, `info`, and `debug`. Trace mode is only available if Fluent Bit was built with the _`WITH_TRACE`_ option enabled. | `info` |
```